src/main/java/com/minecolonies/entity/ai/minimal/EntityAIGoHome.java

```diff
@@ -58,4 +73,23 @@ public void startExecuting()
 
         citizen.getNavigator().tryMoveToXYZ((double) pos.getX() + 0.5D, (double) pos.getY(), (double) pos.getZ() + 0.5D, 1.0D);
     }
+
+    /**
+     * While going home play a goHome sound for the specific worker by chance.
+     */
+    private void playGoHomeSounds()
+    {
+        Random rand = new Random();
+
+        int chance = rand.nextInt(ONE_IN_A_THOUSAND);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'chance' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)